packages/gasket-plugin-config/lib/plugin.d.ts

```diff
@@ -0,0 +1,17 @@
+import type { GasketConfig, HookExecTypes, MaybeAsync } from '@gasket/engine';
```

Comment:
Several of these `.d.ts` files have unused imports for GasketConfig and/or HookExecTypes. My IDE picks up on it, but is that, expected? Should we go ahead and introduce eslint settings for `.ts`?

Reply:
I was under the impression that you had to import them before you could extend but I'm not 100% sure on that. Let me review and see if I can remove the unused imports.

Reply:
Looks like those are not necessary after all; removed.